uniffi_bindgen/src/bindings/kotlin/templates/CallbackInterfaceTemplate.kt

```diff
@@ -20,18 +20,25 @@ public interface {{ type_name }} {
 // The ForeignCallback that is passed to Rust.
 internal class {{ foreign_callback }} : ForeignCallback {
     @Suppress("TooGenericExceptionCaught")
-    override fun invoke(handle: Handle, method: Int, args: RustBuffer.ByValue): RustBuffer.ByValue {
+    override fun invoke(handle: Handle, method: Int, args: RustBuffer.ByValue, outBuf: RustBufferByReference): Int {
         val cb = {{ ffi_converter }}.lift(handle) ?: throw InternalException("No callback in handlemap; this is a Uniffi bug")
         return when (method) {
-            IDX_CALLBACK_FREE -> {{ ffi_converter }}.drop(handle)
+            IDX_CALLBACK_FREE -> {
+                {{ ffi_converter }}.drop(handle)
+                0
```

Comment:
These magic values are a bit jarring - maybe drop a comment pointing at the docs for ForeignCallback?